README.md

```diff
@@ -18,24 +20,12 @@ PHP 5.5 and later
 ## Installation & Usage
 ### Composer
 
-To install the bindings via [Composer](http://getcomposer.org/), add the following to `composer.json`:
+To install the bindings via [Composer](http://getcomposer.org/), do:
 
 ```
-{
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/danskernesdigitalebibliotek/ddb-cover-service-php-client.git"
-    }
-  ],
-  "require": {
-    "danskernesdigitalebibliotek/ddb-cover-service-php-client": "*@dev"
-  }
-}
+composer require composer require danskernesdigitalebibliotek/ddb-cover-service-php-client
```

Comment:
That may be a compose require to much :-)

Reply:
Fixed